<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterOptionsDialog</class>
 <widget class="QDialog" name="FilterOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Elevation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBox_Elevation">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>90</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Azimuth</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBox_Azimuth">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SNR(C/No)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QSpinBox" name="spinBox_SNRCNo">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_OK">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_Apply">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
